models/models.py

```diff
@@ -0,0 +1,5 @@
+from pydantic import BaseModel
+
+class add_values(BaseModel):
```

Comment:
As you can see `BaseModel` follow a `camel case`. The class name follow camel case wehereas the variable names follow `snake case`